ingestion/src/metadata/ingestion/source/database/dbt/dbt_service.py

```diff
@@ -229,6 +240,18 @@ def process_dbt_descriptions(self, data_model_link: DataModelLink):
         Method to process DBT descriptions using patch APIs
         """
 
+    @abstractmethod
+    def yield_dbt_glossary(self, data_model_link: DbtObjects):
+        """
+        Method to patch DBT glossary
+        """
+
+    @abstractmethod
+    def yield_dbt_tier(self, data_model_link: DbtObjects):
```

Comment:
qq: why do we need a special tier step? Don't we have a yield_tags already there? Same with glossary, in the end it's just a tag with a different source. Should we be able to just reuse the same method but yielding extra details?